integration/setup/setup.go

```diff
@@ -94,6 +95,18 @@ type SetupOpts struct {
 
 	// SetupUser true creates a user and returns an authenticated client.
 	SetupUser bool
+
+	// Options to override default backend configuration.
+	BackendOptions *BackendOpts
+}
+
+// BackendOpts represents backend configuration used for test setup.
+type BackendOpts struct {
+	// Capped collections cleanup interval.
+	CappedCleanupInterval time.Duration
+
+	// Percentage of documents to cleanup for capped collections.
```

Comment:
Let's document default value 20 if not set.

Reply:
Done.